flask/app/analyses.py

```diff
@@ -303,6 +318,10 @@ def create_analysis():
         updated_by_id=updated_by_id,
         datasets=found_datasets,
     )
+
+    if request.json.get("priority"):
```

Comment:
I think to keep things consistent with the other POST endpoints, you can add to 312-319 (where an analysis object is instantiated), `priority = request.json.get('priority')` which will default to none thanks to the get method if no priority field is supplied. Since the enum check occurs before any value must be valid.

Reply:
:+1: updated in [e4b9593](https://github.com/ccmbioinfo/stager/pull/522/commits/e4b9593025382ef80a15513aeece84099c059f1e#diff-dfde89d18ccd5b96d88d26c96d8893c69183bcb28d98a401425b603b891d7701R315)